cloudflare/resource_cloudflare_access_group_test.go

```diff
@@ -0,0 +1,359 @@
+package cloudflare
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+var (
+	accountID   = "01a7362d577a6c3019a474fd6f485823"
+	email       = "test@example.com"
+	accessGroup cloudflare.AccessGroup
+)
+
+func TestAccCloudflareAccessGroupConfig_Basic(t *testing.T) {
+	name := "cloudflare_access_group.test"
```

Comment:
instead of "test", let's use a random string from [`generateRandomResourceName()`](https://github.com/terraform-providers/terraform-provider-cloudflare/blob/d526f8e6ef39db23d2189398d8b61b31179ac115/cloudflare/provider_test.go#L77-L79) to prevent any overlap.